modules/openthread/Kconfig.thread

```diff
@@ -162,3 +162,7 @@ config OPENTHREAD_CLI_TCP_ENABLE
 
 config OPENTHREAD_HISTORY_TRACKER
 	bool "History tracker support"
+
+config OPENTHREAD_MAC_STAY_AWAKE_BETWEEN_FRAGMENTS
+	bool "Stack speed optimisation for large packet"
+	default y if OPENTHREAD_IP6_FRAGM
```

Comment:
Agree to have a Kconfig way to set this define, but not sure it should be enabled by default since it would affect power consumption for battery powered devices.